### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by mykola-koval**
[DEMO LINK](https://mykola-koval.github.io/react_static-list-of-posts/)

### CODE REVIEW

**Review by vladyslav-harkusha — APPROVED** [review]

File: src/components/CommentList/CommentList.tsx [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+import { CommentInfo } from '../CommentInfo/CommentInfo';
+
+interface Props {
+  comments: CommentToPost[],
+}
+
+export const CommentList: React.FC<Props> = ({ comments }) => (
+  <ul className="card card-content">
+    Comments:
+    {comments.map(comment => <CommentInfo comment={comment} />)}
```
Comment (Elena-brv) [inline src/components/CommentList/CommentList.tsx:11]: add key

File: src/components/PostList/PostList.tsx [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+interface Props {
+  posts: Post[],
+}
+
+export const PostList: React.FC<Props> = ({ posts }) => (
+  <ul className="content container is-fluid">
+    {posts.map(post => (
+      <li key={post.id} className="card card-content">
+        <PostInfo post={post} />
+        <CommentList comments={post.comments} />
```
Comment (Elena-brv) [inline src/components/PostList/PostList.tsx:14]: CommentsList should be inside PostInfo

**Review by Elena-brv — CHANGES_REQUESTED** [review]

File: src/components/CommentList.tsx [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+import { CommentInfo } from './CommentInfo';
+
+interface Props {
+  comments: CommentToPost[],
+}
+
+export const CommentList: React.FC<Props> = ({ comments }) => (
+  <ul className="card card-content">
+    Comments:
+    {comments.map(comment => <CommentInfo key={comment.id} comment={comment} />)}
```
Comment (Elena-brv) [inline src/components/CommentList.tsx:11]:
```suggestion
    {comments.map(comment => (
      <CommentInfo key={comment.id} comment={comment} />
    ))}
```

File: src/components/UserInfo.tsx [hunk 4]
```diff
@@ -0,0 +1,9 @@
+import React from 'react';
+
+interface Props {
+  user: User,
+}
+
+export const UserInfo: React.FC<Props> = ({ user }) => (
+  <p className="has-text-right">{`posted by @${user.username} (${user.name}) <${user.email}>`}</p>
```
Comment (Elena-brv) [inline src/components/UserInfo.tsx:8]:
```suggestion
  <p className="has-text-right">
    {`posted by @${user.username} (${user.name}) <${user.email}>`}
  </p>
```

**Review by Elena-brv — APPROVED** [review]

**Pull request closed without merging**